Sources_v3/StreamChatUI/ChatChannel/ChatFileAttachmentListView+ItemView.swift

```diff
@@ -0,0 +1,179 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+extension ChatFileAttachmentListView {
+    open class ItemView: View, UIConfigProvider {
+        public var attachment: _ChatMessageAttachment<ExtraData>? {
+            didSet { updateContentIfNeeded() }
+        }
+
+        public var tapHandler: () -> Void = {}
+
+        public var actionIconSize = CGSize(width: 20, height: 20)
+
+        // MARK: - Subviews
+
+        public private(set) lazy var fileIconImageView: UIImageView = {
+            let imageView = UIImageView()
+            imageView.contentMode = .scaleAspectFit
+            return imageView.withoutAutoresizingMaskConstraints
+        }()
+
+        public private(set) lazy var fileNameLabel: UILabel = {
+            let label = UILabel()
+            label.font = UIFont.preferredFont(forTextStyle: .body).bold
+            label.adjustsFontForContentSizeCategory = true
+            return label.withoutAutoresizingMaskConstraints
+        }()
+
+        public private(set) lazy var fileSizeLabel: UILabel = {
```

Comment:
I think our codebase will benefit from:
```swift
extension UILabel {
        public static func labelForTextStyle(_ style: UIFont.TextStyle) -> UILabel {
             let label = UILabel()
             label.font = UIFont.preferredFont(forTextStyle: style)
             label.adjustsFontForContentSizeCategory = true
             return label.withoutAutoresizingMaskConstraints
         }
}
```

Usage:
```swift
public private(set) lazy var fileSizeLabel: UILabel = .labelForTextStyle(.footnote)
```

Reply:
Will do in separate PR for all UI components 👍